docs/extensibility/available-filters.md

```diff
@@ -243,6 +241,26 @@ __experimentalRegisterCheckoutFilters( 'automatic-coupon-extension', {
 |---|---|
 | <img src="https://user-images.githubusercontent.com/5656702/123768988-bc55eb80-d8c0-11eb-9262-5d629837706d.png" /> | ![image](https://user-images.githubusercontent.com/5656702/124126048-2c57a380-da72-11eb-9b45-b2cae0cffc37.png) |
 
+### Hide a snackbar notice containing a certain string
+Let's say we want to hide all notices that contain the string `auto-generated-coupon`.
+
+We would do this by setting the value of the `snackbarNoticeVisibility` to false for the notices we want to hide.
```

Comment:
**nitpic** Coming from support, I tend not to use pronouns (_we_, _you_, etc.), but would write instructions neutral.  Instead of 

> Let's say we want to hide all notices that contain the string `auto-generated-coupon`.
> We would do this by setting the value of the `snackbarNoticeVisibility` to false for the notices we want to hide.

I'd write

> To hide all notices that contain the string `auto-generated-coupon`, the value of the `snackbarNoticeVisibility` need to be set to false, for the notices that should be hidden.

That said, I see that other sections use a similar wording. Hence, my prefix _nitpic_.

Reply:
Thanks Niels, this is a good thing to keep in mind! I'll update these instances in this PR, but not the others. Instead, I will create a cooldown issue to review documentation and re-write it in a neutral way without using personal pronouns.

Reply:
That sounds like a good approach. I would also only touch the section I highlighted and not any other section as this would exceed the scope of this change. I'll mark this feedback as resolved as well.